modules/core/src/main/java/org/apache/ignite/internal/ClusterMetricsMXBeanImpl.java

```diff
@@ -112,7 +112,7 @@ private ClusterMetrics metrics() {
 
     /** {@inheritDoc} */
     @Override public float getBusyTimePercentage() {
-        return metrics().getBusyTimePercentage() * 100;
+        return metrics().getBusyTimePercentage();
     }
 
     /** {@inheritDoc} */
```

Comment:
Did you think about including boundaries of this value in mx bean description in `ClusterMetricsMXBean`?

Reply:
based on the name of the method, yes